packages/drng/dispatcher.go

```diff
@@ -4,27 +4,22 @@ import (
 	"errors"
 	"time"
 
-	"github.com/iotaledger/goshimmer/packages/binary/drng/payload"
-	"github.com/iotaledger/goshimmer/packages/binary/drng/payload/header"
-	"github.com/iotaledger/goshimmer/packages/binary/drng/subtypes/collectivebeacon"
-	"github.com/iotaledger/goshimmer/packages/binary/drng/subtypes/collectivebeacon/events"
-	cb "github.com/iotaledger/goshimmer/packages/binary/drng/subtypes/collectivebeacon/payload"
 	"github.com/iotaledger/hive.go/crypto/ed25519"
 	"github.com/iotaledger/hive.go/marshalutil"
 )
 
 // Dispatch parses a DRNG message and process it based on its subtype
-func (drng *DRNG) Dispatch(issuer ed25519.PublicKey, timestamp time.Time, payload *payload.Payload) error {
-	switch payload.Header.PayloadType {
-	case header.TypeCollectiveBeacon:
+func (drng *DRNG) Dispatch(issuer ed25519.PublicKey, timestamp time.Time, payload *Payload) error {
```

Comment:
Single character for method receiver here.
```suggestion
func (d *DRNG) Dispatch(issuer ed25519.PublicKey, timestamp time.Time, payload *Payload) error {
```